.gitignore

```diff
@@ -58,7 +57,4 @@ hs_err_pid*
 replay_pid*
 
 !java-agent/benchmark/releases/*.jar
-/smoke-tests/play/.gradle/
```

Comment:
Why?

Reply:
I made .gradle a non-absolute path above which seems like low chance of collision :)